reference/test/matrix/diagonal_kernels.cpp

```diff
@@ -388,4 +413,46 @@ TYPED_TEST(DiagonalComplex, MtxIsConjugateTransposable)
 }
 
 
+TYPED_TEST(DiagonalComplex, InplaceAbsolute)
+{
+    using Diag = typename TestFixture::Diag;
+    using value_type = typename TestFixture::value_type;
+
+    auto exec = gko::ReferenceExecutor::create();
```

Comment:
nit: newline too much:
```suggestion
    using value_type = typename TestFixture::value_type;
    auto exec = gko::ReferenceExecutor::create();
```